reduction/Imaging_instructions.txt

```diff
@@ -72,3 +72,40 @@ of the important scripts (i.e., this directory:
 https://github.com/ALMA-IMF/reduction/tree/master/reduction).
 
 When you create mask region files, they belong in $ALMAIMF_ROOTDIR/clean_regions/.
+
+
+Additional tricks of cleaning in selfcal
+=============================================
+Be conservative in making the first image before the first iteration of the selfcal.
+That is, clean relatively relatively shallow probably by setting the threshold to be
+4 times rms or a bit greater. Otherwise, some artefacts may be introduced into the final
+image. In the later iterations (e.g., from iter2, 3, ...,) of selfcal, the threshold can be
+constrained down to 2 times rms.
+
+Scales in band3 could be set larger than those in the band 6, which is demonstrated for the 
+source G353.41, for example, scales = [0, 3, 9, 27] in band3 but scales = [0, 3, 9] in band6. 
+Note that the scales are ought to be in a reasonable range, i.e., smaller than
+or comparable to the size the largest mask, otherwise, the final image could be distorted.
+
+In each selfcal, the threshold and the region mask could be changeable due to the rms varying
+with each selfcal. So, it is recommend to change them following your previous criteria 
+on the threshold and region mask. 
```

Comment:
I don't quite understand this.  What do you mean "could be changeable"?

Would you be comfortable with this rephrasing:
"The region mask and thresholds within the region should be changed at each iteration.  Specifically, you should generally increase the area included in the mask with each subsequent iteration."